example_graph_test.go

```diff
@@ -104,13 +98,9 @@ func ExampleGraphNew_tls() {
 
 	graph := redisgraph.GraphNew("social", pool.Get())
 
-	q := "CREATE (w:WorkPlace {name:'RedisLabs'})"
-	res, err := graph.Query(q)
-
-	q = "MATCH (w:WorkPlace) RETURN w"
-	res, err = graph.Query(q)
+	q := "CREATE (w:WorkPlace {name:'RedisLabs'}) RETURN w"
+	res, _ := graph.Query(q)
 
-	res.Next()
```

Comment:
in previous calls to `graph.Query(q)` there's a follow up with `res.Next()` before calling `r := res.Record()`
I think calling `Next` is mandatory to get a record.

Reply:
thank you Roi. the others I knew were working but this one ( since we don't use the Output ( for now )) had this. 
Also corrected the redisgraph_tls_client.go example with the short version of the query